src/g_editor.c

```diff
@@ -4831,6 +4987,12 @@ void g_editor_setup(void)
         A_GIMME, A_NULL);
     class_addmethod(canvas_class, (t_method)canvas_motion, gensym("motion"),
         A_FLOAT, A_FLOAT, A_FLOAT, A_NULL);
+#ifdef HAVE_KEYBOARDNAV
+    class_addmethod(canvas_class, (t_method)canvas_goto, gensym("goto"),
+        A_FLOAT, A_NULL);
+    class_addmethod(canvas_class, (t_method)canvas_objindexes, gensym("objindexes"),
+        A_FLOAT, A_NULL);
```

Comment:
the plural of "index" is "indices".
also, the name is misleading (it doesn't get nor set the object-indices). a better name is probably "showindices" (iirc, it also shows the indices of non-objects, like commens, messageboxes,...)